src/core/credential_format/mod.rs

```diff
@@ -213,6 +216,10 @@ pub enum ClaimFormatDesignation {
     /// the Credential format can be utilized with any type of Credential (or mdoc document types).
     #[serde(rename = "mso_mdoc")]
     MsoMDoc,
+    /// The format is defined by IETF Web Authorization Protocol (https://datatracker.ietf.org/doc/draft-ietf-oauth-selective-disclosure-jwt/)
+    /// which defines a Selective Disclosure for JWTs (SD-JWT)
+    #[serde(rename = "sd_jwt_vc")]
+    SdJwtVc,
```

Comment:
I suggested using `spruceid_vcdm2_sd_jwt` as a name for this variant (`spruceid_` to make it clear it's a custom extension). I'll admit this might have been a bit excessive :sweat_smile: but with this name I'm affraid of a collision with an eventual future official addition to the DIF registry, and also that the `jwt_vc` part could be conflated with the legacy ["JWT-VC" format from the VCDM1.1 spec](https://www.w3.org/TR/vc-data-model/#json-web-token). What do you think @cobward?

Reply:
Agreed, let's err on the side of over-specifying. I would also be open to not including this in the OID4VP library. We can just use the `Other` variants since this isn't a standard value.

Reply:
I think you're right, I'm in favor of just adding an `Other` variant and delegate the implementation of the VCDM2.0 JOSE COSE SD-JWT profile as an application specific extension, implemented in the application.